reactor-test/src/main/java/reactor/test/publisher/TestPublisher.java

```diff
@@ -72,13 +75,43 @@
 	 * subscribers. It caches the {@link #next(Object)} events and replays them to
```

Comment:
Let's use `buffers` instead of `caches`. Also, rephrase the `and replays...` part to something more along the lines of:
> and tracks how many elements have been seen by each subscriber in order to correctly replay the buffer to each subscriber, with backpressure.

Reply:
fixed in `polish: javadoc, formatting and null detection` 84d0afe